src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1174,92 +1170,97 @@ private static function get_wp_parent_theme(): array {
 			$auto_updates = (array) get_site_option( 'auto_update_themes', array() );
 		}
 
-		$active_theme  = wp_get_theme();
+		$active_theme = wp_get_theme();
 		$parent_theme = $active_theme->parent();
 
-		if ( $parent_theme ) {
-			$parent_theme_version       = $parent_theme->version;
-			$parent_theme_version_debug = $parent_theme_version;
+		if ( ! $parent_theme ) {
+			return array(
+				'label'  => __( 'Parent Theme' ),
+				'fields' => array(),
+			);
```

Comment:
are you sure adding this redundancy is beneficial?

Reply:
the thought behind the early-abort is to avoid mistakes based on humans needing to continue parsing the function. before this was extracted it was pre-set to be an `array()` and the `if` was purely additive. now, with the logic localized, we have to do something affirmative in each branch.

I don't see any one solution purely dominant here. the redundancy doesn't bother me at all, and I tend to be more concerned about `if` complexity. either way, I've reverted this change for a different fix in 276de8bae6 which pre-sets the `$fields = array()` var so it's not unset when returning